misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/util/CodeCoverageUtils.java

```diff
@@ -70,8 +75,13 @@ public static void unzipCompiledSource(Path source, Path destination, String org
                 }
 
             }
+
             copyClassFilesToBinPath(destination, destJarDir, orgName, moduleName, version);
             deleteDirectory(new File(destJarDir));
+        } catch (NoSuchFileException e) {
+            String msg = "Source file for " + moduleName + " doesnt exist. Cannot generate Code coverage";
```

Comment:
Let's improve the error message a bit.
"Unable to geenerate code coverage. Source file for the module " + moduleName + "does not exist."
WDYT?

Reply:
Yes. The way it is currently worded doesnt sound nice. 

I think it would be better if I change it to `"Unable to generate code coverage for the module " + moduleName ". Source file does not exist"`. If we word it like this, then it implies that it couldn't generate the coverage only for this particular module and that it will continue to do so for the other modules. 

WDYT?

Reply:
Changed it to "Unable to generate code coverage for the module " + moduleName ". Source file does not exist"